lib/companies_house/client.rb

```diff
@@ -19,6 +22,7 @@ def initialize(config)
       @endpoint = URI(config[:endpoint] || ENDPOINT)
       @open_timeout = config[:open_timeout] || 60
       @read_timeout = config[:read_timeout] || 60
+      @instrumentation = default_instrumentation
```

Comment:
Do you want to allow people to pass an instrumenter via `:instrumentation` in `config`? I don't really mind since it looks like it can be overridden later but it might make for a more consistent API?

```ruby
client = CompaniesHouse::Client.new(...)
client.instrumentation = ...
```

vs

```ruby
client = CompaniesHouse::Client.new(instrumentation: ...)
```

Reply:
fixed that 👍